tests/jballerina-unit-test/src/test/resources/test-src/types/finaltypes/final-field-test-negative.bal

```diff
@@ -71,3 +71,9 @@ function testLocalFinalValueWithoutTypeInitializedFromFunction() {
 function getName() returns string {
     return "Ballerina";
 }
+
+listener http:MockListener ml = new http:MockListener(8080);
+
+public function testListenerVariable() {
```

Comment:
Shall we improve the name of the function to reflect what's actually being tested?

Reply:
done